### DISCUSSION TITLE
Personal Meeting of Developers

### DISCUSSION THREAD

**Discussion by klaus-halfmann** [widelands/widelands#4982] [Ideas]
This is a specal kind of Request.

I would like to invite / organise all developers for a personal
meeting / Workshop / Get Together. I would organize some
rooms in a youth hostel and reserve some accomodation
for a weekend. This would be in Germany, near frankfurt.

This should be an informal  meeting - for now - so we get 
to know each other, have some fun _and_ discuss the
widelands future. But we can setup up some rought agenda, 
too.

I assume some developer can pay for a roome themselfes. 
In case this is to expensive for someone please contact me 
via PM on widelands.org (Hasi50)

Does this sound like a good idea? If I get > 2 positive responses.
I will try to fix some date, even this year.

**Comment by hessenfarmer** [comment]
May I request to choose the location keeping in mind a good network connection and booking a meeting room.

**Comment by klaus-halfmann** [comment]
> May I request to choose the location keeping in mind a good network connection and booking a meeting room.

Sure! Since I got 2 replies I will go ahead an book a room for >= 5 Persons an reserve 4 rooms.
This should bee a weekend starting late Friday and a complete Saturday.
@Noordfrees I think we will need you as a core developer, too.

AFAIK this can happen even this year. Bigger events will need a reservation a year ahead.

**Comment by klaus-halfmann** [comment]
@matthiakl would you be able to attend such an Event in Germany? (I thik we will have some video tool anyway)

**Comment by Noordfrees** [comment]
I won't be able to participate unfortunately as I'm usually too busy on weekends. Perhaps I could look in via video call or such but I'd probably only know whether even this would be possible for me very shortly beforehand.

Edit: But I can already say with 99% certainty that I will have no time at all on the 11.09. :(

**Comment by klaus-halfmann** [comment]
I got a reservation for Fr. 10.09 till Su. 12.09..2021 will do some personal checks, next.
OK we have to fix this till 07.07.21. I would not invite "Gamers" only but perhaps
someone for the translators.I would really like if  @gunchleoc or @SirVer could attend
as ouf "Founding Parents".

We will have a room for >=5 Persons with a fixed line Internet access.

Address is:

Jugendherberge Bad Homburg
Mühlweg 17
61348 Bad Homburg

Tel.:+49 6172 23950 
Fax: +49 6172 22312 

Please contact me in person for details.

**Comment by matthiakl** [comment]
I will be able to attend personally

**Comment by klaus-halfmann** [comment]
@hessenfarmer can you cofirm this date?
I reserved 5 rooms so 3 additional people would  be OK.

**Comment by klaus-halfmann** [comment]
@frankystone what about you?

Which additonal Developers / Translators could be interested?

**Comment by frankystone** [comment]
I think about participation, but need some time. Is there a deadline for commitment?

**Comment by klaus-halfmann** [comment]
I must confirm this latest 7.7.2021. So it wold it woud be fine fi you could decide till then

**Comment by hessenfarmer** [comment]
Unfortunately I can't confirm this date until the the 7th of july.    
Maybe we should first get all developers / translators intrested in such a meeting. then try to fix a date (using some sort of a poll and then find a location. Cause there  might be more locations then suitable dates from my experience.

**Comment by hessenfarmer** [comment]
Perhaps this should be moved from the issues to discussions as well   
Anyhow a big thank you to Klaus for the idea and the organisation and I feel a bit guilty of not beeing able to confirm the date.

**Comment by SirVer** [comment]
Great idea! Reminds me of the wihacks we used to have back in the 2010s. I will unfortunately not be able to join in easily at this time in my life :(

**Comment by frankystone** [comment]
> Maybe we should first get all developers / translators intrested in such a meeting. then try to fix a date (using some sort of a poll and then find a location.

:+1: 

For finding dates one can use [doodle](https://doodle.com/)

**Comment by klaus-halfmann** [comment]
My idea was to have this as a kind of "pre"-meeting, to get to know each other.
A Big-Meeting woud then be planned (with Doodle and everything) next year.
I think we must get bette organised, first.

**Comment by frankystone** [comment]
Klaus, can you please close this issue and start a [Discussion](https://github.com/widelands/widelands/discussions) with this topic?

I think the idea is very good, but it is not an issue :-)

**Comment by klaus-halfmann** [comment]
For some reason this already is (or has become?) a [Discussion](https://github.com/widelands/widelands/discussions).
I will try to extend the reservation for another week until @hessenfarmer may fix the date.

For now please give me some answers about the following questions:
* We all agree, that we want to _play_ widelands, too. (nobody will be forced, however)
* How long in advance do you need to plan such an Event? 3 Month , 6 Month, a year, longer
* Is Fr Evening till Sunday ok? with main Events on Saturday
* Shall we also invite: Translators, Grafic Supporters, Players, ... ?
* Is a youth hostle ok? (Hotel Steigenberger is on the other side, but I will not sponsor that)
* I am willing to sponspor some of the common costs, anybody else?.
* I would try to get some sponsors, but I am lousy in doing so.

Sorry for the late response, My job and other issues are eating my time.

**Reply by SirVer** [reply]
Late reply: If this is close to Munich and I am not traveling at the time, I would love to come by for a few hours or a full day. I am also willing to sponsor such an event with some money. I probably would need 6 weeks in advance to know if I can arrange it or not.

I am most interested in meeting people and understanding what is currently going on with Widelands, playing is lower on my priority list.

**Comment by hessenfarmer** [comment]
Late reply as well 
I am not against playing some rounds.
However I am more interested in getting to know other developers and discuss issues that we face which will give us problems.
Obtaining some common goals for next versions are a desirable side effect.    
I am willing to sponsor as well, but I would prefer some loacation with single rooms and some privacy. In big cities some business hotels offer quite good rates for the weekends normally as they are making their money from  monday to friday.     
having Half a year in advance would be desirable for my planning.    

Thanks to Klaus for trying to organise this.      
@SirVer as you seem to be located in Munich too, I would love to  meet you for a beer (which I will surely pay)  to get a glue in the history of widelands.

**Reply by SirVer** [reply]
@hessenfarmer Beers sounds lovely, but finding time is quite hard for me these days. Usually I am better reachable in the mornings or for lunch during working hours. Let's find some time that works unilaterally, feel free to fire me email (preferred) or PM on the homepage.

**Comment by klaus-halfmann** [comment]
OK lets try something else first: I will keep some jitsi meeting open every Friday 19:00 https://meet.ffmuc.net/WidelandsChat

Meanwhile I have set up my fist Doodle and ask for some Days / Weekends next year.

https://doodle.com/poll/g86nvyzre5ruspdq?utm_source=poll&utm_medium=link

I once was in the youth hostel in munic and it was quite OK.
I would stay there but maybe @hessenfarmer has some better ideas?

Should we invite non-developers / players, too?
